HLTrigger/Configuration/python/HLT_2024v14_cff.py

```diff
@@ -9334,7 +9334,14 @@
       DR_Veto_E = cms.double( 0.07 ),
       NoiseTow_EB = cms.double( 0.04 ),
       Noise_EE = cms.double( 0.1 ),
-      UseRecHitsFlag = cms.bool( False ),
+      #UseRecHitsFlag = cms.bool( False ),
```

Comment:
you should not edit this file directly, but write a customization function in `HLTrigger/Configuration/python/customizeHLTforCMSSW.py`

Reply:
Okay sorry, let me fix it